cache/clock_cache.h

```diff
@@ -9,8 +9,434 @@
 
 #pragma once
 
+#include <array>
+#include <memory>
+#include <string>
+
+#include "cache/cache_key.h"
+#include "cache/sharded_cache.h"
+#include "port/lang.h"
+#include "port/malloc.h"
+#include "port/port.h"
 #include "rocksdb/cache.h"
+#include "rocksdb/secondary_cache.h"
+#include "util/autovector.h"
+#include "util/distributed_mutex.h"
 
-#if defined(TBB) && !defined(ROCKSDB_LITE)
-#define SUPPORT_CLOCK_CACHE
+#if !defined(ROCKSDB_LITE)
+#define SUPPORT_CLOCK_CACHE  // TODO(Guido) Do we need this?
```

Comment:
It can be removed, because we no longer have an external dependency (TBB) controlling whether it's supported, and it's not used in the public API